presto-hive/src/main/java/io/prestosql/plugin/hive/metastore/thrift/ThriftMetastoreUtil.java

```diff
@@ -570,6 +566,24 @@ public static OptionalLong fromMetastoreNullsCount(long nullsCount)
         return OptionalLong.of(nullsCount);
     }
 
+    public static HiveColumnStatistics fromMetastoreNumFalse(BooleanColumnStatsData booleanStatsData)
```

Comment:
s/fromMetastoreNumFalse/createBooleanColumnStatistics

Reply:
@kokosing Thanks for your comment;
 You mean I should replace the method name `fromMetastoreNumFalse` with new name `createBooleanColumnStatistics`? There already exists a method `createBooleanColumnStatistics` in class `HiveColumnStatistics`.

Reply:
good point, please inline `fromMetastoreNumFalse` method then. It is not used anywhere else.